tests/module_test.py

```diff
@@ -0,0 +1,11 @@
+import pytest
+
+from honeybee_3dm.room import to_room
+from honeybee_3dm.face import to_face
+from honeybee_3dm.togeometry import mesh_to_face3d
+
+
+def test_import():
+    assert to_room
+    assert to_face
+    assert mesh_to_face3d
```

Comment:
unless we have some better tests to check, I think we can remove this file. These tests are not really useful.